app/controllers/verify/phone_controller.rb

```diff
@@ -9,7 +9,10 @@ def new
     end
 
     def create
-      if step.complete
+      result = step.submit
+      analytics.track_event(Analytics::IDV_PHONE_CONFIRMATION, result.to_h)
```

Comment:
I think there will be exceptions where we want to manage the keys for analytics by hand (`result.except(:redirect_url)` being a great example) but I think that for most cases, we just want to hand the entire object in to our analytics, and we should optimize for that case.

I think it still makes sense to move the `.to_h` inside, but I don't feel that strongly about it.